app/scripts/controllers/projectBrowseCatalog.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .controller('ProjectBrowseCatalogController',
+    function($scope,
+             $routeParams,
+             DataService,
+             APIService,
+             AuthorizationService,
+             Catalog,
+             CatalogService,
+             Navigate,
+             NotificationsService,
+             ProjectsService) {
+      var currentProjectName = $routeParams.project;
+      var imageStreamsVersion = APIService.getPreferredVersion('imagestreams');
+
+
+      ProjectsService.get(currentProjectName).then(function() {
+        AuthorizationService.getProjectRules(currentProjectName).then(function () {
+          var canIAddToProject = AuthorizationService.canIAddToProject(currentProjectName);
+          if (!canIAddToProject) {
+            Navigate.toProjectOverview(currentProjectName);
+            return;
+          }
+
+          // Show all catalog items, project templates, and project image streams
+          CatalogService.getCatalogItems().then(function (items) {
+            Catalog.getProjectCatalogItems(currentProjectName, false, true).then( _.spread(function(catalogServiceItems, errorMessage) {
+              DataService.list(imageStreamsVersion, {namespace: currentProjectName}, function(isData) {
```

Comment:
Pass `true` for `includeImages` to `getProjectCatalogItems` instead?

https://github.com/openshift/origin-web-catalog/blob/master/src/services/catalog.service.ts#L116-L125

Reply:
The CatalogService passes true to Catalog if i!sTemplateServiceBrokerEnabled

Reply:
> The CatalogService passes true to Catalog if i!sTemplateServiceBrokerEnabled

I think you might be looking at a different method / argument? Unless I'm misreading the code, there's one for image streams for the `getProjectCatalogItems` method you're calling directly. You're passing `false` just above.

Reply:
Yeah, I was looking at the CatalogService.getCatalogItems call.

However, the Catalog.getProjectCatalogItems defaults the includeImages and includeTemplates to true:

`  public getProjectCatalogItems(projectName: string, includeImages: boolean = true, includeTemplates: boolean = true, partialObjectMetadataList: boolean = false ) {
`

so I think this is OK, right? When testing I do see both project templates and project image streams when browsing in project context which I do not see from the landing page.